source/compatibility.tex

```diff
@@ -42,6 +42,19 @@
 
 \rSec2[diff.cpp20.expr]{\ref{expr}: expressions}
 
+\diffref{expr.prim.id.unqual}
+\change
+Change move-eligible \grammarterm{id-expression}s from lvalues to xvalues.
+\rationale
+Simplify the rules for implicit move.
+\effect
+Valid \CppXX{} code that relies on a returned \grammarterm{id-expression}'s
```

Comment:
```suggestion
Valid \CppXX{} code that relies on a returned \grammarterm{id-expression}
```

Reply:
I think the existing text is fine English grammar, but I'll defer to the editor @tkoeppe for that.

Reply:
I agree, both are fine, and no reason to deviate from the approved wording.

Reply:
Is this supposed to imply "_id-expression_'s value category"?

Reply:
No, I think it's fine as is. "X's being a Y" is the same as "the fact that X is a Y", so you can think of this as "the id-expression is an lvalue", which seems fine - either "an expression is an lvalue" or "an expression's value category is 'lvalue'", but the former seems definitely fine, and common.

Reply:
I see. Thank you.